### PR TITLE
fix: add unhandled return error handling.

### PR DESCRIPTION
**Pull request by zemyblue**
### Add unhandled return error handling.
* baseapp/grpcserver.go
  * grpc.SetHeader(grpcCtx, md)
* server/start.go
  * serverCtx.Viper.BindPFlags(cmd.Flags())
* server/util.go
  * serverCtx.Viper.BindPFlags(cmd.Flags())
  *  serverCtx.Viper.BindPFlags(cmd.PersistentFlags())
* x/auth/client/rest/query.go
  *  packStdTxResponse(w, clientCtx, txRes)
* x/bank/client/cli/tx.go
  * cmd.Flags().Set(flags.FlagFrom, args[0])
* x/feegrant/client/cli/tx.go
  * cmd.Flags().Set(flags.FlagFrom, args[0])
* x/foundation/keeper/member.go
  *  info.SetDecisionPolicy(policy)
* x/slashing/genesis.go
  * keeper.AddPubkey(ctx, consPk)
* x/slashing/keeper/hooks.go
  * keeper.AddPubkey(ctx, consPk)
* x/staking/genesis.go
  * keeper.SetValidatorByConsAddr(ctx, validator)
* x/staking/keeper/msg_server.go
  * k.SetValidatorByConsAddr(ctx, validator)
* x/wasm/client/cli/query.go
  * flagSet.Set(flags.FlagPageKey, string(raw))

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/foundation/keeper/member.go [hunk 1]
```diff
@@ -16,7 +16,10 @@ func validateMetadata(metadata string, config foundation.Config) error {
 
 func (k Keeper) UpdateDecisionPolicy(ctx sdk.Context, policy foundation.DecisionPolicy) error {
 	info := k.GetFoundationInfo(ctx)
-	info.SetDecisionPolicy(policy)
+	err := info.SetDecisionPolicy(policy)
+	if err != nil {
```
Comment (0Tech) [inline x/foundation/keeper/member.go:20]: You can use this one liner, because it returns only `error`.
```suggestion
	if err := info.SetDecisionPolicy(policy); err != nil {
```
It has the great advantage of not creating a new variable (outside of `if`). Then, you may revert the changes caused by the naming collision in variables.

**Review by 0Tech — APPROVED** [review]
LGTM

**Review by tkxkd0159 — APPROVED** [review]

File: baseapp/grpcserver.go [hunk 2]
```diff
@@ -61,7 +61,9 @@ func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
 		grpcCtx = context.WithValue(grpcCtx, sdk.SdkContextKey, sdkCtx)
 
 		md = metadata.Pairs(grpctypes.GRPCBlockHeightHeader, strconv.FormatInt(height, 10))
-		grpc.SetHeader(grpcCtx, md)
+		if err = grpc.SetHeader(grpcCtx, md); err != nil {
```
Comment (0Tech) [inline baseapp/grpcserver.go:64]: There is practically no issue, but it would be better not to change the existing variable.
```suggestion
		if err := grpc.SetHeader(grpcCtx, md); err != nil {
```
It may also apply to other lines.

**Review by dudong2 — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -44,6 +44,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Bug Fixes
 * (x/foundation) [\#772](https://github.com/line/lbm-sdk/pull/772) export x/foundation pool
 * (baseapp) [\#781](https://github.com/line/lbm-sdk/pull/781) implement method `SetOption()` in baseapp
+* (global) [\#782](https://github.com/line/lbm-sdk/pull/782) add unhandled return error handling
+
```
Comment (0Tech) [inline CHANGELOG.md:48]:
```suggestion
```

**Review by 0Tech — COMMENTED** [review]
;)

**Comment by zemyblue** [comment]
In this PR, most error handling codes can't do unittest. So I'll skip the codecov error of ci in this case.
